.env.example

```diff
@@ -24,13 +24,16 @@ REACT_APP_HOTJAR_ID=
 REACT_APP_INTERCOM_APP_ID=
 REACT_APP_ROLLBAR_ACCESS_TOKEN=
 REACT_APP_DELIGHTED_API_KEY=
+REACT_APP_MARKETPLACE_GRAPH_URL=
+REACT_APP_IPFS_URL=
 
 # Contracts
 
 REACT_APP_MANA_TOKEN_CONTRACT_ADDRESS=0x2a8fd99c19271f4f04b1b7b9c4f7cf264b626edb
 REACT_APP_LAND_REGISTRY_CONTRACT_ADDRESS=0xf87e31492faf9a91b02ee0deaad50d51d56d5d4d
 REACT_APP_ESTATE_REGISTRY_CONTRACT_ADDRESS=0x959e104e1a4db6317fa58f8295f586e1a978c297
-
+REACT_APP_ENS_CONTRACT_ADDRESS=0x112234455c3a32fd11230c42e7bccd4a84e02010
+REACT_APP_RESOLVER_CONTRACT_ADDRESS=0x12299799a50340FB860D276805E78550cBaD3De3
```

Comment:
We should add these 4 new env vars to the `.ci/.env.dev`, `.ci/.env.stg` and `.ci/.env.prod` files